### PR TITLE
repo sync

### PR DESCRIPTION
**Pull request by Octomerger**
This is an automated pull request to sync changes between the public and private repos.

:robot: This pull request should be merged (not squashed) to preserve continuity across repos, so please let a bot do the merging!

### CODE REVIEW

**Comment by walid161hatam** [comment]
‫در چهارشنبه ۲ دسامبر ۲۰۲۰‏، Octomerger Bot‏ <notifications@github.com>
نوشت:‏

File: content/github/setting-up-and-managing-organizations-and-teams/managing-security-and-analysis-settings-for-your-organization.md [hunk 1]
```diff
@@ -9,34 +9,47 @@ versions:
...
+
+1. Go to the security and analysis settings for your organization. For more information, see "[Displaying the security and analysis settings](#displaying-the-security-and-analysis-settings)."
+1. In the "{% data variables.product.prodname_dependabot %} repository access" section, click the settings button **{% octicon "gear" aria-label="The Gear icon" %}**. 
+   ![Repository access setting button](/assets/images/help/organizations/repository-access-cog-button.png)
+   A list is displayed showing all of the private repositories in your organization.
+   ![The Repositories list](/assets/images/help/organizations/repositories-dialog.png)
+1. Select the repositories that {% data variables.product.prodname_dependabot %} can access.
+1. Click **Select repositories**.
```
Comment (MASTR-cmd) [inline content/github/setting-up-and-managing-organizations-and-teams/managing-security-and-analysis-settings-for-your-organization.md:50]: Thank you

File: content/github/setting-up-and-managing-organizations-and-teams/managing-security-and-analysis-settings-for-your-organization.md [hunk 2]
```diff
@@ -9,34 +9,47 @@ versions:
 ### About management of security and analysis settings
```
Comment (BFREEZ) [inline content/github/setting-up-and-managing-organizations-and-teams/managing-security-and-analysis-settings-for-your-organization.md:9]: Shannon bowman

File: content/github/setting-up-and-managing-organizations-and-teams/managing-security-and-analysis-settings-for-your-organization.md [hunk 3]
```diff
@@ -9,34 +9,47 @@ versions:
...
-{% endif %}
-
-### Enabling or disabling features for existing repositories
+### Displaying the security and analysis settings
 
 {% data reusables.profile.access_profile %}
 {% data reusables.profile.access_org %}
 {% data reusables.organizations.org_settings %}
 {% data reusables.organizations.security-and-analysis %}
-5. Under "Configure security and analysis features", to the right of the feature, click **Disable all** or **Enable all**.
```
Comment (BFREEZ) [inline content/github/setting-up-and-managing-organizations-and-teams/managing-security-and-analysis-settings-for-your-organization.md:24]: Enable

**Review by BFREEZ — APPROVED** [review]

**Pull request merged**